web/scripts/display_policies.js

```diff
@@ -0,0 +1,134 @@
+const BASE_URL="http://localhost:8080"
```

Comment:
can we read this port from env var `KUBERNETS_SERVER_PORT`, just like we are doing in go server? https://github.com/infracloudio/kubernets/blob/master/cmd/serve/server.go#L17

Reply:
reading env env in vanilla JS is not that easy, we can look into that later.

Reply:
sounds good